app/src/main/res/values/string_constants.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`strings_constants` is too generic, it sounds like a bucket for everything. That would lead to a mess in the future. I suggest to keep one file with URLs, for other thing we will create a separate file. 
Suggestion is to rename file to `urls.xml` and drop "Link" from the string name

Reply:
Done